src/main.rs

```diff
@@ -101,18 +117,20 @@ fn show_program(s: &mut Cursive) {
   ];
 
   let mut table = TableView::<Programpost, BasicColumn>::new()
-    .column(BasicColumn::Tidspunkt, "Name", |c| {
+    .column(BasicColumn::Tidspunkt, "Tidspunkt", |c| {
       c.width_percent(5).align(HAlign::Center)
     })
-    .column(BasicColumn::Tittel, "Count", |c| c.width_percent(10))
-    .column(BasicColumn::Beskrivelse, "Rate", |c| c);
+    .column(BasicColumn::Tittel, "Happening", |c| c.width_percent(10))
+    .column(BasicColumn::Beskrivelse, "Beskrivelse", |c| c);
 
   table.set_items(programposter);
+  table.set_on_submit(show_programpost_details);
```

Comment:
Siden vi har programpostene tilgjengelig her, kunne vi ha gjort noe ala
```suggestion
  table.set_on_submit(|s_, _row, index| show_programpost_details(s_, programposter[index]));
```
da med
```
fn show_programpost_details(s: &mut Cursive, post: Programpost)
```
?